data-prepper-plugins/elasticsearch/src/main/java/com/amazon/dataprpper/plugins/sink/elasticssarch/aws/interceptor/http/AwsRequestSigningApacheInterceptor.java

```diff
@@ -0,0 +1,227 @@
+/*
+ * Copyright 2012-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package com.amazon.dataprpper.plugins.sink.elasticssarch.aws.interceptor.http;
```

Comment:
Typo: `dataprpper`. The file is also in the wrong location.

The unit test is also in the wrong package.